README.md

```diff
@@ -1,17 +1,139 @@
-# Registers Ruby Client
+# GOV.UK Registers Ruby Client
+
+This documentation is for developers interested in using this Ruby client to integrate their service with GOV.UK Registers. 
+
 ## Installation
 In your Gemfile add:
 ```
 gem 'registers-ruby-client', git: 'https://github.com/openregister/registers-ruby-client.git'
 ```
 ## Usage
+
+_Note: examples use the `country` register_
+
 ```
 require 'registers_client'
 
 registers_client = RegistersClient::RegistersClientManager.new({ cache_duration: 3600 })
+
+
 register_data = registers_client.get_register 'country', 'beta'
-register_data.get_records.first[:item]
- => {"citizen-names"=>"Soviet citizen", "country"=>"SU", "end-date"=>"1991-12-25", "name"=>"USSR", "official-name"=>"Union of Soviet Socialist Republics"} 
- ```
+```
+
+### Methods
+
+#### `get_entries`
+
+Example usage:
+
+`register_data.get_entries.first[:item]`
+
+Expected output:
+
+```
+{"citizen-names"=>"Soviet citizen", "country"=>"SU", "end-date"=>"1991-12-25", "name"=>"USSR", "official-name"=>"Union of Soviet Socialist Republics"}
+get_records
+```
+
+#### `get_records`
+
+Example usage:
+
+`register_data.get_records.first[:item]`
+
+Expected output:
+
+```
+{"citizen-names"=>"Soviet citizen", "country"=>"SU", "end-date"=>"1991-12-25", "name"=>"USSR", "official-name"=>"Union of Soviet Socialist Republics"}
+```
+
+#### `get_metadata_records`
+
+Example usage:
+
+`register_data.get_metadata_records.first[:item]`
+
+Expected output:
+
+`{"name"=>"country"}`
+
+#### `get_field_definitions`
+
+Example usage:
+
+`register_data.get_field_definitions.first[:item]`
+
+Expected output:
+
+```
+{"cardinality"=>"1", "datatype"=>"string", "field"=>"country", "phase"=>"beta", "register"=>"country", "text"=>"The country's 2-letter ISO 3166-2 alpha2 code."}
+```
+
+#### `get_register_definition`
+
+Example usage:
+
+`register_data.get_register_definition.to_json`
+
+Expected output:
+
+```
+"{\"key\":\"register:country\",\"entry_number\":229,\"timestamp\":\"2016-08-04T14:45:41Z\",\"hash\":\"sha-256:610bde42d3ae2ed3dd829263fe461542742a10ca33865d96d31ae043b242c300\",\"item\":{\"fields\":[\"country\",\"name\",\"official-name\",\"citizen-names\",\"start-date\",\"end-date\"],\"phase\":\"beta\",\"register\":\"country\",\"registry\":\"foreign-commonwealth-office\",\"text\":\"British English-language names and descriptive terms for countries\"}}"
+```
+
+#### `get_custodian`
+
+Example usage:
+
+`register_data.get_custodian.to_json`
```

Comment:
I think `register_data.get_custodian[:item]['custodian']` is a better example to show how to read the custodian's name.

Reply:
I agree and have done this in f4eaa04ac550a00a8020d61b7f61baef2158aefd but why do we not do the same in e.g.

`puts register_data.get_metadata_records.first[:item]['name']`